test/systemtests/util_test.go

```diff
@@ -1452,6 +1462,16 @@ func (s *systemtestSuite) SetUpTestVagrant(c *C) {
 			PvtSubnet:        "172.19.0.0/16",
 		}), IsNil)
 		time.Sleep(120 * time.Second)
+	} else {
+		c.Assert(s.cli.GlobalPost(&client.Global{FwdMode: "bridge",
+			Name:             "global",
+			NetworkInfraType: "default",
+			Vlans:            "1-4094",
+			Vxlans:           "1-10000",
+			ArpMode:          "proxy",
+			PvtSubnet:        "172.19.0.0/16",
+		}), IsNil)
+		time.Sleep(120 * time.Second)
```

Comment:
same as above, and why 2 minute sleep vs 40s as above?

Reply:
done

Reply:
i guess it was because it's faster on metal server